### ISSUE TITLE
[ Mint 21 and Cinnamon 5.4.12] Option "Always on top" does not always works

### ISSUE THREAD

**Issue by Pat-Ibulaire** [linuxmint/cinnamon#11322]
```
 * Cinnamon version (cinnamon --version)
--> Cinnamon 5.4.12
   - Please specify if you are using the daily builds PPA
     (https://launchpad.net/~linuxmint-daily-build-team/+archive/ubuntu/daily-builds).
--> No
 * Distribution - (Mint 21, Arch, Fedora 37, etc...)
--> See screenshot attached
 * Graphics hardware *and* driver used
--> See screenshot attached
 * 32 or 64 bit
--> 64 bit
 * Attach ~/.xsession-errors, or /var/log/syslog
```
![2022-12-07_09-58](https://user-images.githubusercontent.com/9960491/206135896-601d3c37-71ba-4f7b-9c1c-d4a537fd6a7f.png)
[xsession-syslog.TXT](https://github.com/linuxmint/cinnamon/files/10174468/syslog.TXT)
[xsession-errors.TXT](https://github.com/linuxmint/cinnamon/files/10174399/xsession-errors.TXT)

### Issue
A window with the "Always on top" option set is not always on top ...
It works sometimes but that regularly fails.

### Steps to reproduce
Just put two windows and set the option on one of them.
See the fullscreen attached for an exemple.
![2022-12-07_09-54](https://user-images.githubusercontent.com/9960491/206135780-92205f4a-2755-4d28-94df-9ac77916cb2d.png)

**Comment by icarter09** [comment]
@Pat-Ibulaire do you have a case or steps where you are able to consistently duplicate the issue? Thanks.

**Comment by Pat-Ibulaire** [comment]
No, you just have to put two windows and switch the focus to test. For me, it doesn't work in most cases. I think you should try it with Cinnamon version 5.4.12.
